### DISCUSSION TITLE
Segment integration still available?

### DISCUSSION THREAD

**Discussion by krejko** [getredash/redash#5926] [Q&A]
On the Redash website, Segment is listed as a supported integration: https://redash.io/integrations/segment

However, even after upgrading our self-hosted on-prem solution to 10.1, we don't see it in the list of data sources. Is this integration still supported? Or was this a cloud-only feature? 

Any info on if or how we can use the segment integration would be very helpful. Thank you!

**Comment by arikfr** [comment]
When I check the Integrations main page they are listed as a partner next to Stitch and Snowplow Analytics. I think this was more about being able to query the data Segment sends into your database/data warehouse.

**Comment by krejko** [comment]
I see. That does make sense from the integrations main page, but it isn't obvious from segment integrations page. Based on your comment, I am guessing there hasn't been a direct integration with Segment; Just the ability to use data you transfers with segment?

**Reply by arikfr** [reply]
Correct.
